app/views/wiki/edit.html.erb

```diff
@@ -21,6 +21,15 @@
       <li><a href="/wiki/authoring-help#Advanced+formatting"><%= t('wiki.edit.advanced_formatting') %></a></li>
     </ul>
   </div>
+  <hr />
+  <div class="sidebar-nav hidden-sm">
+    <p class="nav-header"><h4><b>Templates</b></h4></p>
+    <p>Use these to start a page from one of these templates:</p>
+    <ul>
+      <li><a href="/post?n=14552">Method Template lite</a></li>
```

Comment:
@Ronaq13 as I mentioned here issue requires the change of wiki template and at `/post` we create new note so please change href to "/wiki/new?n=14552" .Thanks.

Reply:
Changed accordingly

Reply:
Line 29 need to be changed as "/post" is on that, line 30 was fine earlier